tox.ini

```diff
@@ -74,7 +74,7 @@ commands = {toxinidir}/scripts/check_copyright_notice.py --directory {toxinidir}
 [testenv:hash_check]
 skipsdist = True
 usedevelop = True
-deps = ipfshttpclient
+deps = ipfshttpclient==0.6.0.post1
```

Comment:
better to stick with one version to avoid surprises.

Reply:
Needs to be properly bumped in pipenv too